src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.cs

```diff
@@ -40,43 +43,67 @@ public object Invoke()
         [DebuggerGuidedStepThrough]
         public object Invoke(object? arg1)
         {
-            object result = _methodBaseInvoker.CreateInstanceWithFewArgs(new Span<object?>(ref arg1));
+            if (_parameterCount > 4)
```

Comment:
Should this be 1?
I agree with Jan, where are the tests especially since this already broke a scenario.

Reply:
Yep thanks